setup.py for entrypoints by Marius Brehler <marbre@linux.sungazer.de>.

Patch by Marius Brehler <marbrbre@linux.sungazer.de>.
--- /dev/null
+++ b/setup.py
@@ -0,0 +1,13 @@
+from distutils.core import setup
+
+setup(name='entrypoints',
+      version='0.2.1',
+      description='Discover and load entry points from installed packages.',
+      author='Thomas Kluyver',
+      author_email='thomas@kluyver.me.uk',
+      url='https://github.com/takluyver/entrypoints',
+      packages=['entrypoints'],
+      classifiers=[
+          'License :: OSI Approved :: MIT License'
+      ]
+)
\ No newline at end of file
